### ISSUE TITLE
_intToBase64 bug causing decryption fail when create key using exponent=65537

### ISSUE THREAD

**Issue by river-littlepay** [appsup-dart/jose#16]
## Issue
Issue was decryption failed when using public key created like codes below.
```
final publicKey = JsonWebKey.rsa(
      modulus: BigInt.parse("12345678"),
      exponent: BigInt.parse("65537"),
    );
```

After troubleshooting, was `_intToBase64` converting `65537` to a wrong base64 encoded string.
```
String _intToBase64(BigInt v) {
  return _bytesToBase64(v
      .toRadixString(16) -----> "10001"
      .replaceAllMapped(RegExp('[0-9a-f]{2}'), (m) => '${m.group(0)},') ----> "10,00,1"
      .split(',')
      .where((v) => v.isNotEmpty)
      .map((v) => int.parse(v, radix: 16))
      .toList());
}
```

Problem was "10001" should be grouped as "1,00,01" not "10,00,1". Tested this problem affect all hex value with odd length.

## Workaround
1. using following code to do int to base64 encoding:
```
String _intToBase64_fixed(BigInt v) {
  return _bytesToBase64(v
      .toRadixString(16)
      .balanced()
      .replaceAllMapped(RegExp('[0-9a-f]{2}'), (m) => '${m.group(0)},')
      .split(',')
      .where((v) => v.isNotEmpty)
      .map((v) => int.parse(v, radix: 16))
      .toList());
}

String _bytesToBase64(List<int> bytes) {
  return base64Url.encode(bytes).replaceAll('=', '');
}

extension Balancing on String {
  String balanced() {
    return this.length % 2 == 0 ? this : "0" + this;
  }
}
```
2. use following to create key
```
var jwk = new JsonWebKey.fromJson(
      {
        "kty": "RSA",
        "n": _intToBase64_fixed(modulus),
        "e": _intToBase64_fixed(exponent),
      },
    );
```

**Comment by rbellens** [comment]
fixed in version `0.3.0-nullsafety.2`

**Comment by river-littlepay** [comment]
@rbellens Great work on keeping this awesome plugin updated, thanks!

BTW, Travis-CI is failing because of dart version. 
https://docs.travis-ci.com/user/languages/dart/
